courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>set up these following code cells with a final Lab Task.</p><p> </p><p><strong>Lab Task</strong>: In the code cells below, use the functions you created above to make bar plots for various model inputs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>